RetailCoder.VBE/UI/SourceControl/SourceControlBindings.cs

```diff
@@ -34,15 +30,13 @@ public override void Load()
             Bind<IDeleteBranchView>().To<DeleteBranchForm>();
             Bind<IMergeView>().To<MergeForm>();
 
-            //presenters
-            Bind<IChangesPresenter>().To<ChangesPresenter>();
-            Bind<IBranchesPresenter>().To<BranchesPresenter>();
-            Bind<ISettingsPresenter>().To<SettingsPresenter>();
-            Bind<IUnsyncedCommitsPresenter>().To<UnsyncedCommitsPresenter>();
+            //factories 
+            // todo: check on note below
+            // ninject is complaining about also having a SourceControlProviderFactoryProxy and a FolderBrowserFactoryProxy
```

Comment:
No. No binding for `ISourceControlProvider` because it used to come from the factory. Looks like I've a bit more work to do here.